### ISSUE TITLE
Issue with ’ character in the filename

### ISSUE THREAD

**Issue by jy95** [clement-escolano/parse-torrent-title#3]
Hello,

I found a strange behaviour with  ’ character in the filename : it just cuts the title.

Demo time : 

``` js
const nameParser = require('parse-torrent-title');
var title = "Maman, j’ai encore raté l’avion.avi";
var json = nameParser.parse(title);
console.log(json);
```

gives :

``` js
{ container: 'avi', title: 'Maman, j’ai encore raté l’' }
``` 

Any ideas how to solve it ? Here is a tap test to help you

```js
it("should return the title", () => {
        const releaseName = "Maman, j’ai encore raté l’avion";
        expect(parse(releaseName)).to.deep.include({ title: "Maman, j’ai encore raté l’avion" });
    });
```

Thanks to fix it.

EDIT: seems this bug already exists in original package (run this sample on https://runkit.com to it)
```js
var tnp = require('torrent-name-parser');
var title = "Maman, j’ai encore raté l’avion.avi";
tnp(title);
```

**Comment by clement-escolano** [comment]
Hi !
Actually this is not an issue with the character `’` but with the container regex which detects `avion` as the `AVI` container.
It should be fixed in the version `1.1.0` of the package. Feel free to close the issue if this is fine for you.
Cheers
PS : I did not forget the issue about the episode name. This is however a big change so it will need more time.

**Comment by jy95** [comment]
Thanks, it looks fixed. Strange that nobody notices this bug in the first project ^^

PS: No problemo for the delay of the episode name. My citation to resume it : 
> Better an Oops than a what If.
FYI : That is an english sentence that means : "at least you tried"
